### PR TITLE
DM-30745: modernize event handling to match new information from Tekniker

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

**Review by wvreeven — APPROVED** [review]
This all looks real god to me.

File: doc/tma_interface.rst [hunk 1]
```diff
@@ -87,6 +87,19 @@ Commands have the following fields, in order:
...
 ^^^^^^^
 
+All replies are in json format and have the following fields:
+
+* id (int): a ReplyId enum value.
+* timestamp (float): the time the message created or sent, as TAI unix.
+* parameters (dict): additional parameters which vary depending on the id.
+
+The vendor's documentation for replies ("events") is
+https://gitlab.tekniker.es/aut/projects/3151-LSST/LabVIEWCode/documentation/pxicontroller_documentation/-/blob/develop/02%20CommandsAndEventsManagement/03%20Events.md
```
[the remark this answers is not in the document]
Comment (r-owen) [reply doc/tma_interface.rst:97]: Yes, alas. But it's the only documentation we have and it's in flux. In the long term we need to find a way to make it public -- either by copying it somewhere or making our own version.

File: python/lsst/ts/MTMount/enums.py [hunk 2]
```diff
@@ -293,6 +288,33 @@ class Source(enum.IntEnum):
     PXI = 100
 
 
+class System(enum.IntEnum):
+    """Values for the system field of several replies, including:
+
+    * POWER_STATE
+    * CHILLER_STATE
+    * MOTION_CONTROLLER_STATE
```
[the remark this answers is not in the document]
Comment (r-owen) [reply python/lsst/ts/MTMount/enums.py:296]: This refers to several types of message from the TMA. These should be enum values in the ReplyId enum. I will try to clarify the doc string and also fix these names if they no longer match ReplyId.

[the remark this answers is not in the document]
Comment (r-owen) [reply python/lsst/ts/MTMount/enums.py]: Here is the new doc string:
```
    """Values for the system field of several replies, including:

    * ReplyId.LIMITS
    * ReplyId.MOTION_CONTROLLER_STATE
    * ReplyId.POWER_STATE
    """
```

**Pull request merged**